exec/java-exec/src/test/java/org/apache/drill/exec/rpc/user/security/TestUserBitSSL.java

```diff
@@ -201,6 +202,7 @@ public void testSSLQuery() throws Exception {
     test("SELECT * FROM cp.`region.json`");
   }
 
+  @Ignore
```

Comment:
Parth, can you please add explanation (`@Ignore("some reason")`) why test is ignored (it would be really helpful in future).

Reply:
Will do.